.evergreen/config.yml

```diff
@@ -2312,13 +2312,14 @@ task_groups:
       - func: fix-absolute-paths
       - func: make-files-executable
       - func: assume-ec2-role
-      - command: shell.exec
+      - command: subprocess.exec
         params:
-          shell: bash
+          binary: bash
           include_expansions_in_env: ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"]
-          script: |
-            ${PREPARE_SHELL}
-            ${DRIVERS_TOOLS}/.evergreen/auth_oidc/setup.sh
+          env:
+            MONGODB_VERSION: "8.0"
+          args:
```

Comment:
Note that human18 fails if we don't run with MONGODB_VERSION 8.0

Reply:
That is tracked by https://jira.mongodb.org/browse/PYTHON-4447, which I updated to reflect today's status.